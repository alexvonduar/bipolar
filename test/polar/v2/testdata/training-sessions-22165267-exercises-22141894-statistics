
	▀╖╫
╙m#AMUbARY"д▀!BУ,B╩╘5B:мл AP≥╛AмлчAJh▌