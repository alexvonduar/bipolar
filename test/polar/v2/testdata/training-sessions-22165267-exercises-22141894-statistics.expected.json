{
    "altitude": [
        {
            "average": [
                43.239337921142578
            ],
            "maximum": [
                45.415752410888672
            ],
            "minimum": [
                40.386489868164062
            ]
        }
    ],
    "cadence": [
        {
            "2": [
                89
            ],
            "maximum": [
                82
            ]
        }
    ],
    "heartrate": [
        {
            "average": [
                167
            ],
            "maximum": [
                189
            ],
            "minimum": [
                139
            ]
        }
    ],
    "speed": [
        {
            "average": [
                10.214273452758789
            ],
            "maximum": [
                14.145825386047363
            ]
        }
    ],
    "stride": [
        {
            "2": [
                142
            ],
            "maximum": [
                104
            ]
        }
    ],
    "temperature": [
        {
            "average": [
                21.574859619140625
            ],
            "maximum": [
                27.850000381469727
            ],
            "minimum": [
                19.350000381469727
            ]
        }
    ]
}
